recipes/shyaml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "shyaml" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80650ebfe6fa2e16083451d515207472d60990c1c15fc0fd607c27077790ac23
+  patches:
+    - cli-args.patch
+
+build:
+  noarch: python
+  number: 0
+  #script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
Guess this line can be dropped (or moved below?).

Reply:
Yep, thanks!  Changes made